tests/scripts/ferret.py

```diff
@@ -0,0 +1,329 @@
+#/usr/bin/env python
+
+# python t.py -f /tmp/vxlan_decap.json -s 192.168.8.1
```

Comment:
Wrong comments here

Reply:
This is a copied file - should we keep that cleanup separate from this PR?

Reply:
I would suggest, though, deleting the original file and adding a symlink to this one so we don't have to worry about them getting out of sync. What do you guys think?

Reply:
There was existing practice of replicating files so I did follow suite. Question to the team, are we not going to get rid of Ansible equivalent? If so, we might copy these helper files to pytest dir and we will have smooth cleanup down the road.

Reply:
> Wrong comments here

This is an existing file. I can help fix the comments.

Reply:
See my below comment. I agree that we should copy them over, but I think rather than leaving the original in place it makes sense to symlink it to the copy we make in the pytest directory.